components/camel-bean/src/main/docs/bean-language.adoc

```diff
@@ -20,7 +20,7 @@ methods.
 The xref:manual::bean-binding.adoc[Bean Binding] rules are used to bind the
 xref:manual::message.adoc[Message] Exchange to the method parameters; so you can
 annotate the bean to extract headers or other expressions such as
-xref:components::xpath-language.adoc[XPath] or xref:components::xquery-language.adoc[XQuery] from the message.
+xref:xpath-language.adoc[XPath] or xref:xquery-language.adoc[XQuery] from the message.
```

Comment:
I don't think these are correct, both of these are in the `components` module. Try:

```shell
$ find . -name xpath-language.adoc -o -name xquery-language.adoc
./docs/components/modules/ROOT/pages/xquery-language.adoc
./docs/components/modules/ROOT/pages/xpath-language.adoc
./components/camel-saxon/src/main/docs/xquery-language.adoc
./components/camel-xpath/src/main/docs/xpath-language.adoc
./catalog/camel-catalog/target/classes/org/apache/camel/catalog/docs/xquery-language.adoc
./catalog/camel-catalog/target/classes/org/apache/camel/catalog/docs/xpath-language.adoc
```

Reply:
Hello Mr.Zoran, I think since bean-language.adoc try to cross-reference xpath-language .adoc and xquery-language.adoc which are in the same module(components) they do not require to specify which module they belong to.
So once I change this and ran link check it didn't report any errors as well, 
may I undo this?

Reply:
Yup, please do.